symbols/arm9.yml

```diff
@@ -724,6 +724,14 @@ arm9:
       length:
         NA: 0x4
         EU: 0x4
+    - name: WONDER_GUMMI_IQ_GAIN
+      address:
+        NA: 0x20A1890
+        EU: 0x20A1E14
+      length:
+        NA: 0x1
```

Comment:
It looks like this value's accessed with `ldrsh`, so let's change the length to `0x2`.